tasks/apt/install.yml

```diff
@@ -45,13 +39,13 @@
     group: root
     mode: 0644
 
-- name: Install OpenJDK 7
+- name: Install OpenJDK 8
   apt:
-    name: openjdk-7-jdk
+    name: openjdk-8-jdk
```

Comment:
I'm a little worried about locking this role to java 8, that might break current users and make it harder to install older versions of jenkins. In order to be able to install older versions of jenkins too, let's move the packages to install into a variable called `jenkins_apt_packages`, similar to the pattern used [here](https://github.com/geerlingguy/ansible-role-apache/blob/master/vars/Debian.yml#L8) and [here](https://github.com/geerlingguy/ansible-role-apache/blob/master/tasks/setup-Debian.yml#L7).

That way the user is free to use any version of java they want (in the same way they can currently choose any version of jenkins they want).

If you'd like to also bump up the default version of jenkins that the role uses then please do, then the default `jenkins_apt_packages` can contain java 8 instead of java 7. If you do this then please ensure the README is up to date too.